docs/design/coreclr/botr/logging.md

```diff
@@ -0,0 +1,64 @@
+
+**(FIXME: Can EventPipe also be consumed directly via ETW tools or are they separate?)**
+
+For basic scenarios, you can use the [`dotnet-counters`](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-counters) tool to monitor events and performance counters the runtime reports through EventPipe. For example, to collect the default counters from a project while running it, you could use `dotnet-counters collect -- dotnet run`.
+
+You can also use the [`dotnet-trace`](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-trace) tool to capture EventPipe events from the runtime on the fly, either saving them to a file or sending them to your console. It provides command line options you can use to filter events by severity level or based on keywords. For example, to capture high-severity GC events only from a project while running it, you could use `dotnet-trace collect --clreventlevel error --clrevents gc -- dotnet run`.
+
+Please see the linked documentation pages for more information on how to use each of the above tools. They have an extensive set of options and some helpful features like time limits, attaching to existing processes, and configurable output formatting.
+
+To emit your very own EventPipe events from within the C++ part of the runtime, you can use the `FireEtwXXX` APIs, or their convenience wrappers within the `ETW::` C++ namespace. You likely will want to create new type(s) of events instead of using existing ones. The events are generated by the [`genEventing.py`](https://github.com/dotnet/runtime/blob/main/src/coreclr/scripts/genEventing.py) script based on the definitions in [`ClrEtwAll.man`](https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/ClrEtwAll.man). **(FIXME: Is the process for adding one documented somewhere?)**
+
+To emit EventPipe events from C# you can **(FIXME: Do what?)**
+
+GC StressLog
+------------
+The StressLog is a tool primarily used to diagnose issues with the garbage collector. It is a circular buffer inside the runtime process that usually does not escape, and most StressLog messages are enabled in retail builds, which makes it useful for troubleshooting GC issues in production scenarios. To enable it, you can set the `DOTNET_StressLog` environment variable to `1` **(FIXME: I think?)**, and you can configure it by setting the `DOTNET_LogFacility`, `DOTNET_LogFacility2` and `DOTNET_LogLevel` environment variables. If you are in a situation where you can't attach a debugger to the process, the `DOTNET_StressLogFilename` environment variable can be used to write the StressLog's contents to a file instead.
```

Comment:
StressLog is used for all sorts of different components in the runtime, not just GC